torch/_inductor/sizevars.py

```diff
@@ -454,6 +453,11 @@ def strideof(name):
             code.writeline(f"{name}_stride = {name}.stride()")
             return f"{name}_stride"
 
+        @functools.lru_cache(None)
```

Comment:
This will persist across different compilations and may be incorrect.

Reply:
It doesn't cache actual storage_offset, it just generates a string querying it.